runner/job_manager.py

```diff
@@ -413,3 +419,12 @@ def delete_resources(self, batch_service_client: batch.BatchExtensionsClient,
             logger.info(
                 "Job: {}. did not complete successfully, Container {} was not deleted.".format(
                     self.job_id, self.storage_info.output_container))
+
+    def update_params_with_values_from_keyvault(self, parameters, keyvault_client_with_url):
+        for parameter, value in parameters.items():
+            stringValue = str(value)[11:-2] #substring the inner value from format {"value": "<value-we-want>"}
+            if len(stringValue) > 0 and stringValue.startswith(_keyVault_param_identifier):
+                secretId = stringValue[len(_keyVault_param_identifier):]  #substring the original value without the keyvault identifier prefix
+                secret_bundle = keyvault_client_with_url[0].get_secret(keyvault_client_with_url[1], secretId, '')
+                parameters[parameter] = secret_bundle.value
```

Comment:
Can you also use pep8, it's similar to style cop

Reply:
PyLint gives 86 flags, only a couple of line-too-longs in my code...